metals/src/main/scala/scala/meta/internal/metals/PopupChoiceReset.scala

```diff
@@ -0,0 +1,66 @@
+package scala.meta.internal.metals
+
+import scala.concurrent.Future
+import org.eclipse.lsp4j.ShowMessageRequestParams
+import org.eclipse.lsp4j.MessageType
+import org.eclipse.lsp4j.MessageActionItem
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.concurrent.ExecutionContext
+
+class PopupChoiceReset(
+    tables: Tables,
+    languageClient: MetalsLanguageClient,
+    doctor: Doctor,
+    reconnectToBuildServer: () => Future[Unit]
+) {
+  import PopupChoiceReset._
+
+  def reset(value: String)(implicit ec: ExecutionContext): Future[Unit] = {
+    val result = if (value == BuildTool) {
+      tables.buildTool.reset()
+      reconnectToBuildServer()
+    } else if (value == BuildImport) {
+      tables.dismissedNotifications.ImportChanges.reset()
+      Future.successful(())
+    } else {
+      Future.successful(())
+    }
+    result.foreach(_ => doctor.executeRefreshDoctor())
+    result
+  }
+
+  def interactiveReset()(implicit ec: ExecutionContext): Future[Unit] = {
+    def choicesParams(): ShowMessageRequestParams = {
+      val params = new ShowMessageRequestParams()
+      params.setMessage(
+        "What choice would you like to reset?"
+      )
+      params.setType(MessageType.Info)
+      params.setActions(
+        List(
+          new MessageActionItem(PopupChoiceReset.BuildTool),
+          new MessageActionItem(PopupChoiceReset.BuildImport)
+        ).asJava
+      )
+      params
+    }
+
+    languageClient
+      .showMessageRequest(choicesParams())
+      .asScala
+      .flatMap { item =>
+        if (item.getTitle() == PopupChoiceReset.BuildTool) {
+          reset(PopupChoiceReset.BuildTool)
+        } else if (item.getTitle() == PopupChoiceReset.BuildImport) {
+          reset(PopupChoiceReset.BuildImport)
+        } else {
+          Future.successful(())
+        }
+      }
+  }
+}
+
+object PopupChoiceReset {
+  final val BuildTool = "buildtool"
```

Comment:
```suggestion
  final val BuildTool = "build tool selection"
```

Reply:
:+1:

Reply:
Because it goes into html "build tool selection" is placed in a link as "build+tool+selection".
That's why I also added: `val value = arg.getAsString().replace("+", " ")`
This way we handle both inputs: `build+tool+selection` and `build tool selection` correctly. (same applies to import build)